pkg/processor/build/builder.go

```diff
@@ -1119,6 +1119,7 @@ func (b *Builder) buildProcessorImage(ctx context.Context) (string, error) {
 			ReadinessTimeoutSeconds: b.platform.GetConfig().GetFunctionReadinessTimeoutOrDefault(
 				b.options.FunctionConfig.Spec.ReadinessTimeoutSeconds),
 			SecurityContext: b.options.FunctionConfig.Spec.SecurityContext,
+			BuildLogger:     b.logger,
```

Comment:
Why here and not when initializing the container builder instance?

Then it won't be different than using `k.logger`

Reply:
@TomerShor because we init in only once in the dashboard, which means that same Kaniko builder is used for many functions